server/bin/gold/test-18.txt

```diff
@@ -0,0 +1,223 @@
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-01_1970.01.01T00.00.42.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/BACKED-UP/tarball_config-01_1970.01.01T00.00.42.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-01_1970.01.01T00.00.42.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/TO-RE-UNPACK/tarball_config-01_1970.01.01T00.00.42.tar.xz
+
+Restored 1 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-01_1970.01.01T00.00.42.tar.xz
+
+
+cp -a /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz.md5 /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/
+md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz.md5
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/BACKED-UP/tarball_config-02_1970.01.01T00.00.41.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/TO-RE-UNPACK/tarball_config-02_1970.01.01T00.00.41.tar.xz
+
+Restored 2 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-01/tarball_config-02_1970.01.01T00.00.41.tar.xz
+
+
+cp -a /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench-local/archive.backup/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5 /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/
+md5sum --check /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz.md5
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/BACKED-UP/tarball_config-00_1970.01.01T00.00.40.tar.xz
+ln -s /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/TO-RE-UNPACK/tarball_config-00_1970.01.01T00.00.40.tar.xz
+
+Restored 3 (0 existing) of (3): /var/tmp/pbench-test-server/test-18/pbench/archive/fs-version-001/ctrl-00/tarball_config-00_1970.01.01T00.00.40.tar.xz
```

Comment:
Is there at test which exercises the pre-existing tarball check code?

Reply:
No, I'll enhance 18/19 to add that.